src/mrnet/network/reaction_generation.py

```diff
@@ -118,9 +118,19 @@ def __init__(
         self,
         input_entries,
         single_elem_interm_ignore=["C1", "H1", "O1", "Li1", "P1", "F1"],
+        electron_free_energy=-2.15,
+        temperature=298.15,
```

Comment:
Should this point to ROOM_TEMP now? Also do we have a standardized default for electron free energy? if not, should we define one like we have for room temp?

Reply:
Yep. I thought I had changed everything before... maybe this came from a merge with Daniel's work? In any case, will fix.

Sent you a private message on electron_free_energy. I think default values make sense, but the defaults that make sense in general don't make sense for our work, and vice versa. This is one of those "are we building this for us, or for the community?" type questions.